components/SolutionsSection.tsx

```diff
@@ -18,7 +18,7 @@ const SolutionsSection: FunctionComponent = ({ ...props }) => {
       >
         <div className="mt-14">
           <SectionTitle
-            title="Soluções"
+            sectionTitle="Soluções"

```

Comment:
This is something that should go in the storybook, not on the component itself (I think)